faq.md

```diff
@@ -0,0 +1,113 @@
+Frequently Asked Questions
+
+## What is TED AI Open Source Hackathon?
+The [TED AI Open Source Hackathon](https://ai-event.ted.com/hackathon) is an exciting programming event that challenges you to find ways to use Generative AI to tackle social and environmental issues. You will have the opportunity to use your coding skills and collaborate with fellow developers, all in the name of good.
```

Comment:
Change the link to https://tedai-sanfrancisco.ted.com/hackathon/

Reply:
Addressed in 75a5b311cac6e25ccad6dade58ace598e559553d